pkg/services/ngalert/notifier/status.go

```diff
@@ -12,5 +14,22 @@ func (am *Alertmanager) GetStatus() apimodels.GettableStatus {
 	if am.ready() {
 		config = am.config.AlertmanagerConfig
 	}
-	return *apimodels.NewGettableStatus(&config)
+	cs := amv2.ClusterStatusStatusDisabled
```

Comment:
Are there any tests for `GetStatus`? Is it easy to add a test covering this and mocking `am.peer`, or would this require significant extra effort?

Reply:
It requires significant effort, as you need to start Grafana in a HA way.

Reply:
That being said, I tested all the changes on my local machine and on my local k8s cluster.